common/djangoapps/util/enterprise_helpers.py

```diff
@@ -16,6 +17,7 @@
     pass
 
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+EC_BRANDING_FILTER_PARAM = 'branding_param'
```

Comment:
let's be more specific with the actual name of this parameter in session -- `branding_param` seems too generic to me.